packages/amplify-category-function/src/provider-utils/awscloudformation/secrets/test.ts

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs');
```

Comment:
should this be fs or fs-extra?

Reply:
lol this whole file was me experimenting with the runtime hook template. It should be removed